app.py

```diff
@@ -1868,23 +1883,37 @@ def logout():
     logout_user()
     return redirect(url_for('index'))
 
+# #########################################################
+@app.route('/downloadFeedBack/<string:project_id>/<string:evaluation_name>/<string:show_score>', methods=['GET', 'POST'])
+@login_required
+def downloadFeedBack(project_id, evaluation_name, show_score):
+
+    project = Permission.query.filter_by(project_id=project_id).first()         #find project
+
+    evaluation_name_find_in_db = Evaluation.query.filter_by(project_owner=current_user.username, project_name=project.project).all()    #look up all evaluations associated with project
+    
+    now = datetime.now().time()
+    current_time = now.strftime("%H_%M_%S")
+    filename = "feedback_{}_{}.zip".format(evaluation_name, current_time)
```

Comment:
I noticed that zip files in the server directory stay around forever. We want them to be temporary files. Now, we can't just delete the zip file immediately, since send_file() depends on it existing. So here's an idea:

Before downloadFeedback creates a new zipObj, look for existing zip files for this user and delete them. That way we never have more than O(1) old zip files on the server.

Also, for cleanliness, lets put the temporary zip file inside of the user's folder

Reply:
Don't we want to look for existing evaluation zip files? Since a zip file contains all html pages of a single evaluation. 
So if a user has 3 evaluations in a project, then he can have 3 zip files. And I would look for existing evaluation zip files and delete them when they want to "re-download" one of the 3 evaluations.
Am I understanding this right?